src/scene/materials/standard-material.js

```diff
@@ -816,6 +816,13 @@ class StandardMaterial extends Material {
             this._setParameter('material_heightMapFactor', getUniform('heightMapFactor'));
         }
 
+        this.useDisplacement = this.displacementFactor !== 0;
```

Comment:
Actually. I think we should leave the flag on the material, but let user set it.
The reason for that is that if the value is set to 0, we do not want to have to compile a new shader because the flag has changed - it should use the same shader with zero displacement. Shader should only be regenerated when the user actually changes the flag on the material.

Reply:
Not much to change here then, just removing the line.